src/test/java/com/mercateo/sqs/utils/visibility/VisibilityTimeoutExtenderTest.java

```diff
@@ -21,22 +27,30 @@
 
 class VisibilityTimeoutExtenderTest {
 
+    private VisibilityTimeoutExtender uut;
+
     @Mock
     private AmazonSQS sqsClient;
 
-    private VisibilityTimeoutExtender uut;
-
     @Mock
     private ErrorHandlingStrategy<?> errorHandlingStrategy;
 
+    private final Retryer<ChangeMessageVisibilityResult> retryer = RetryerBuilder
+            .<ChangeMessageVisibilityResult>newBuilder()
+            .retryIfExceptionOfType(SdkClientException.class)
```

Comment:
There is a high probability that the retry configuration differs in the test. I think we only want to change the Wait- & StopStrategy in the test ==> move the retrier into VisibilityTimeoutExtender, see also [review suggestions  ](https://github.com/Mercateo/sqs-utils/pull/19/commits/230e31b360c31e5a2c8ff3502cfea33ea1805728#diff-1eba36121d426a3d75d856aff4785c9166919a374541962987767d47769b93f7R58)